ansible_runner/streaming.py

```diff
@@ -0,0 +1,216 @@
+import codecs
+import io
+import json
+import os
+import stat
+import sys
+import tempfile
+import uuid
+import zipfile
+
+import ansible_runner
+import ansible_runner.plugins
+from ansible_runner import utils
+
+
+class UUIDEncoder(json.JSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, uuid.UUID):
+            return obj.hex
+        return json.JSONEncoder.default(self, obj)
+
+
+# List of kwargs options to the run method that should be sent to the remote executor.
+remote_run_options = (
+    'forks',
+    'host_pattern',
+    'ident',
+    'ignore_logging',
+    'inventory',
+    'limit',
+    'module',
+    'module_args',
+    'omit_event_data',
+    'only_failed_event_data',
+    'playbook',
+    'verbosity',
```

Comment:
If I read this right, this allow-listing process is still going to block remote containerized jobs.

Reply:
Yeah, after looking at the attrs on `RunnerConfig` quite a bit yesterday, I'm thinking that most of them need to be passed on.

Reply:
Since you removed the `worker_in` `worker_out` kind of options, I think that _all_ the kwargs should be passed.

Reply:
Yeah, I think you are right.